Makefile

```diff
@@ -64,9 +64,8 @@ user-typing:
 
 # (override)
 github_actions:
-	${PYTHON_EXE} -m venv ${VENV_NAME}
-	${VENV_NAME}/${VENV_BINDIR}/${PYTHON_EXE} -m pip install --upgrade pip
-	${VENV_NAME}/${VENV_BINDIR}/${PYTHON_EXE} -m pip install -e '.[dev]'
+	${PYTHON} -m pip install --upgrade pip
+	${PYTHON} -m pip install -e '.[dev]'
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick (non-blocking):** ‏The other Makefile actions use `$(PYTHON)` syntax, there's no difference (iirc) but it would be nice to stick to one style.